permissions_validators/src/public_blockchain/mod.rs

```diff
@@ -23,6 +25,54 @@ pub static ACCOUNT_ID_TOKEN_PARAM_NAME: Lazy<Name> =
 pub static ASSET_DEFINITION_ID_TOKEN_PARAM_NAME: Lazy<Name> =
     Lazy::new(|| Name::from_str("asset_definition_id").expect("This should never panic"));
 
+/// Enum listing preconfigured permission tokens
+#[derive(IntoSchema)]
+pub enum DefaultPermissionToken {
```

Comment:
```suggestion
pub enum PredefinedPermissionToken {
```

or something similar. I don't think `Default` conveys the proper meaning here

Reply:
also, check our `FromVariant` derive macro

Reply:
>  I don't think `Default` conveys the proper meaning here
True, will change.